testnet/stacks-node/src/config.rs

```diff
@@ -48,6 +48,7 @@ const INV_REWARD_CYCLES_TESTNET: u64 = 6;
 
 #[derive(Clone, Deserialize, Default, Debug)]
 pub struct ConfigFile {
+    pub path: Option<String>,
```

Comment:
Because this field is set at runtime, instead of via `toml::from_str()`, could you call it something that is unlikely to ever be set in the config file?  Something like `__path` instead of just `path` would work.